pkg/plugin/test-keeper/metrics.go

```diff
@@ -0,0 +1,72 @@
+package testkeeper
+
+import (
+	"github.com/arquillian/ike-prow-plugins/pkg/log"
+	gogh "github.com/google/go-github/github"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/arquillian/ike-prow-plugins/pkg/server"
+)
+
+const (
+	// WithTests is a label used in prometheus metrics pull_requests_total for pull request containing some tests.
+	WithTests = "with_tests"
+	// WithoutTests is a label used in prometheus metrics pull_requests_total for pull request without any tests.
+	WithoutTests = "without_tests"
+)
+
+var (
+	pullRequestsCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "pull_requests_total",
+		Help: "Total number of pull requests.",
+	}, []string{"full_name", "type", "plugin_name"})
```

Comment:
I am not sure why it's needed to have plugin name to be added. If you can see in following screenshot you can easily see for which plugin this particular metrics belongs to

![screenshot from 2018-06-19 11-10-39](https://user-images.githubusercontent.com/7941617/41578801-a60fb918-73b1-11e8-9544-a810e53caad2.png)

This screenshot captured for all hosts. When you select for single host, you can see that particulat metrics. 

If particular metric (`pull_requests_total`) is only for single host, then graphana will show only for that host which is the case here for `pull_requests_total` and `okWithoutTestsPullRequest`

Reply:
so, you are saying that Graphana doesn't show these test-keeper metrics when there is no filtering set - in other words, when ![obrazek](https://user-images.githubusercontent.com/1510709/41590745-0ad09a1a-73b8-11e8-9ca9-855c1d72093b.png) ?

Reply:
I didn't mean that. When you have `host: All`, and you are looking for test keeper metrics e.g. `pull_requests_total` (which is only for `test-keeper` not for `wip` and `pr-sanitizer`), so in this case only test-keeper `pull_requests_total` metrics will show.

If we have `host:All`, and we are looking for `total_handled_events` which is there for all plugins it will show all three in graph, and if you hover over it you can easily see for which plugin it belongs to